keyboards/viktus/minne/minne.h

```diff
@@ -0,0 +1,139 @@
+/* Copyright 2023 Viktus Design LLC
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+#define ___ KC_NO
+
+#define LAYOUT_all( \
```

Comment:
most of these could/should be moved to the info.json file. 

if needed, `qmk info -kb viktus/minne -f json` can help with generating the json config for the layouts.

Reply:
Can you explain "most of these"? Is it not required to put all available layouts in *.h anymore? If so would I just put the "LAYOUT_all" in minne.h and everything else defined in info.json?

Just unsure as it wasn't something I saw and trying to correctly implement it as I have more than just this board to correct if so.

Also, The following is one of the layouts from that command and I was wondering a few things:

1) Can ` "filename": "keyboards/viktus/minne/minne.h" `, be removed? Or is that a required part?
2) Am I able to change the format a bit so each switch on a row of the keyboard is a row in this format? Just to help visualize each row a bit easier like how *.h was able to do.
3) Does this now mean qmk.fm/converter is incorrect and out of date? I have been using that to convert the layouts for the JSON (what was used for what is in the JSON now) and wanted to confirm if I should stop using that at least for now.

```
"LAYOUT_all": {
            "c_macro": true,
            "filename": "keyboards/viktus/minne/minne.h",
            "layout": [
                { "label": "K10", "matrix": [1, 0], "w": 1, "x": 0, "y": 0 },
                { "label": "K11", "matrix": [1, 1], "w": 1, "x": 1, "y": 0 },
                { "label": "K01", "matrix": [0, 1], "w": 1, "x": 2, "y": 0 },
                { "label": "K02", "matrix": [0, 2], "w": 1, "x": 3, "y": 0 },
                { "label": "K12", "matrix": [1, 2], "w": 1, "x": 4, "y": 0 },
                { "label": "K13", "matrix": [1, 3], "w": 1, "x": 5, "y": 0 },
                { "label": "K03", "matrix": [0, 3], "w": 1, "x": 6, "y": 0 },
                { "label": "K04", "matrix": [0, 4], "w": 1, "x": 7, "y": 0 },
                { "label": "K14", "matrix": [1, 4], "w": 1, "x": 8, "y": 0 },
                { "label": "K15", "matrix": [1, 5], "w": 1, "x": 9, "y": 0 },
                { "label": "K05", "matrix": [0, 5], "w": 1, "x": 10, "y": 0 },
                { "label": "K06", "matrix": [0, 6], "w": 1.75, "x": 11, "y": 0 },
                { "label": "K30", "matrix": [3, 0], "w": 1.25, "x": 0, "y": 1 },
                { "label": "K31", "matrix": [3, 1], "w": 1, "x": 1.25, "y": 1 },
                { "label": "K21", "matrix": [2, 1], "w": 1, "x": 2.25, "y": 1 },
                { "label": "K22", "matrix": [2, 2], "w": 1, "x": 3.25, "y": 1 },
                { "label": "K32", "matrix": [3, 2], "w": 1, "x": 4.25, "y": 1 },
                { "label": "K33", "matrix": [3, 3], "w": 1, "x": 5.25, "y": 1 },
                { "label": "K23", "matrix": [2, 3], "w": 1, "x": 6.25, "y": 1 },
                { "label": "K24", "matrix": [2, 4], "w": 1, "x": 7.25, "y": 1 },
                { "label": "K34", "matrix": [3, 4], "w": 1, "x": 8.25, "y": 1 },
                { "label": "K35", "matrix": [3, 5], "w": 1, "x": 9.25, "y": 1 },
                { "label": "K25", "matrix": [2, 5], "w": 1, "x": 10.25, "y": 1 },
                { "label": "K26", "matrix": [2, 6], "w": 1.5, "x": 11.25, "y": 1 },
                { "label": "K50", "matrix": [5, 0], "w": 1.75, "x": 0, "y": 2 },
                { "label": "K51", "matrix": [5, 1], "w": 1, "x": 1.75, "y": 2 },
                { "label": "K41", "matrix": [4, 1], "w": 1, "x": 2.75, "y": 2 },
                { "label": "K42", "matrix": [4, 2], "w": 1, "x": 3.75, "y": 2 },
                { "label": "K52", "matrix": [5, 2], "w": 1, "x": 4.75, "y": 2 },
                { "label": "K53", "matrix": [5, 3], "w": 1, "x": 5.75, "y": 2 },
                { "label": "K43", "matrix": [4, 3], "w": 1, "x": 6.75, "y": 2 },
                { "label": "K44", "matrix": [4, 4], "w": 1, "x": 7.75, "y": 2 },
                { "label": "K54", "matrix": [5, 4], "w": 1, "x": 8.75, "y": 2 },
                { "label": "K55", "matrix": [5, 5], "w": 1, "x": 9.75, "y": 2 },
                { "label": "K45", "matrix": [4, 5], "w": 1, "x": 10.75, "y": 2 },
                { "label": "K46", "matrix": [4, 6], "w": 1, "x": 11.75, "y": 2 },
                { "label": "K71", "matrix": [7, 1], "w": 1, "x": 1.38, "y": 3 },
                { "label": "K61", "matrix": [6, 1], "w": 1, "x": 2.38, "y": 3 },
                { "label": "K62", "matrix": [6, 2], "w": 1.25, "x": 3.38, "y": 3 },
                { "label": "K72", "matrix": [7, 2], "w": 1.25, "x": 4.63, "y": 3 },
                { "label": "K73", "matrix": [7, 3], "w": 1, "x": 5.88, "y": 3 },
                { "label": "K63", "matrix": [6, 3], "w": 1.25, "x": 6.88, "y": 3 },
                { "label": "K64", "matrix": [6, 4], "w": 1.25, "x": 8.13, "y": 3 },
                { "label": "K74", "matrix": [7, 4], "w": 1, "x": 9.38, "y": 3 },
                { "label": "K75", "matrix": [7, 5], "w": 1, "x": 10.38, "y": 3 }
            ]
        },
```